src/Umbraco.Core/Media/UploadAutoFillProperties.cs

```diff
@@ -74,8 +75,11 @@ public void Populate(IContentBase content, ImagingAutoFillUploadField autoFillCo
                     {
                         using (var filestream = _mediaFileManager.FileSystem.OpenFile(filepath))
                         {
-                            var extension = (Path.GetExtension(filepath) ?? "").TrimStart(Constants.CharArrays.Period);
-                            var size = _imageUrlGenerator.IsSupportedImageFormat(extension) ? (ImageSize?)_imageDimensionExtractor.GetDimensions(filestream) : null;
+                            var extension = (Path.GetExtension(filepath) ?? string.Empty).TrimStart(Constants.CharArrays.Period);
+                            var size = _imageUrlGenerator.IsSupportedImageFormat(extension)
+                                ? _imageDimensionExtractor.GetDimensions(filestream) ?? (Size?)new Size(Constants.Conventions.Media.DefaultSize, Constants.Conventions.Media.DefaultSize)
+                                : null;
+
```

Comment:
I think we should put this into a private method since it's used again in the other overload, DRY and all that 😉

Reply:
We should indeed keep it DRY 👍🏻